node/src/chain_spec.rs

```diff
@@ -15,36 +15,65 @@ use sc_service::{ChainType, Properties};
 use serde::{Deserialize, Serialize};
 use sp_authority_discovery::AuthorityId as AuthorityDiscoveryId;
 use sp_consensus_babe::AuthorityId as BabeId;
-use sp_core::{crypto::Ss58Codec as _, sr25519, Pair, Public};
+use sp_core::{crypto::UncheckedInto, sr25519, Pair, Public};
 use sp_finality_grandpa::AuthorityId as GrandpaId;
 use sp_runtime::{
 	traits::{IdentifyAccount, Verify},
 	Perbill, SaturatedConversion,
 };
 
 mod testnet {
-	pub const VALIDATORS: &[&str] = &[
-		"5G7DPfMuai9jUokYhxhiPMZ1RTtSAbY8bpHns4LUxB2Jw92p", // Validator 1
-		"5CRmQzLgjknmzRHvgBFZw9mt4q1YgiS5nAcnqM7kRohTi4EG", // Validator 2
-		"5H9CGZS3WJdew9dSLCxHk8ffqgkjdJVecozH6GHwQ7edtWZX", // Validator 3
-	];
-	pub const TECHNICAL_COMMITTEE: &[&str] = &[
-		"5GqLfdcNekxmyDnM1jg7fs2HLXDkDXdvy6UP9pJNSFvaWefm", // TC 1
-		"5Fh71D2b95sro2ZwXZULJMYy7FMst6PAgbYQd8MBifKkrbLx", // TC 2
-		"5GeYbpM2uU35z8kR6bH3fXpRg6Q182WAuBBh843LZVv86ciN", // TC 3
-	];
-	pub const SUDO: &str = "5Hgw5p7Z2MtDiPPsBwJMghngdmmKeqLjWsNiSHqn8YHrehG7";
-	pub const ACCOUNTS: &[&str] = &[
-		"5HdWFbYTMdWQNeQGhHiKNJQiYrfjxdaJ24fg7K4KYbzzPHXD", // Controller validator 1.
-		"5G6EzSquYhJe6pLw7CLtXFciovVvzF6cHDiHL1A74knHdACt", // Controller validator 2.
-		"5GRT695LWMxf9ra37s7L4akM7i4imvFhnNFn9G4fTehUvHXT", // Controller validator 3.
-		"5Ca7t9GrLun6PihvjzeMo2awnaNm7J9udfEBxy7QDTX8q54W",
-	];
-	pub const STAKERS: &[(&str, &str)] = &[
-		(VALIDATORS[0], ACCOUNTS[0]),
-		(VALIDATORS[1], ACCOUNTS[1]),
-		(VALIDATORS[2], ACCOUNTS[2]),
-	];
+	use hex_literal::hex;
+
+	use super::{AccountId, Balance, InitialAuthority, UncheckedInto, AVL};
+
+	pub fn authorities() -> Vec<InitialAuthority> {
+		vec![
+			// Validator 1.
+			InitialAuthority::new(
```

Comment:
Do we have a way of dynamically adding new validators ATM?

Reply:
It already exists, and this code is just for the Genesis Configuration.
I.e. in our [DevNet](https://devnet-avail.polygon.technology/), Validator 4 (5DfSYLzpHvoeoGf8vKjjbUXoTAFrHQTq6VhufycMtxTzjE2m)  and 5(5E24qmjDjh4WPJzyewcYRtEqCgARR3VDfkgwqdg7rG2x1PCH) were added a couple of days after network was started.